src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  width: 200px;
+  background-image: url(../images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-position: center 32px;
+  background-size: 160px auto;
+  padding: 206px 16px 16px;
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+
+  &__info {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+    text-align: left;
+    padding-top: 8px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    margin: 16px 0;
+
+    &--reviews {
+      font-family: Roboto;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 8px 0;
+
+    &--how {
+      font-size: 12px;
+      line-height: 18px;
+      color: #616070;
+    }
+
+    &--what {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__buy {
+    &--button {
+      width: 166px;
+      height: 40px;
+      margin: auto;
+      border-radius: 5px;
+      opacity: 0px;
```

Comment:
The 'opacity' property is set to '0px', which is not a valid value. Opacity should be a number between 0 and 1, without units. Consider changing 'opacity: 0px;' to 'opacity: 0;'.